chat-widget/src/common/Constants.ts

```diff
@@ -64,7 +64,7 @@ export class Constants {
         "application/vnd.microsoft.card.oauth",
     ];
     public static readonly maxUploadFileSize = "500000";
-    public static readonly imageRegex = /(\.)(jpeg|jpg|jiff|png|gif|bmp)$/i;
+    public static readonly imageRegex = /(\.)(jpeg|jpg|jiff|png|gif|bmp|webp)$/i;
```

Comment:
Is this for Image Upload ? Is yes, can we add support for AMR as well. There is an ICM for which repair item is created.

Reply:
This is to restore a previous PR as-is. Not introducing new logic. Please create new work item to address the new issues